test/azlinux_test.go

```diff
@@ -1502,17 +1499,7 @@ func testCustomLinuxWorker(ctx context.Context, t *testing.T, targetCfg targetCo
 		// Now build again with our custom worker
 		// Note, we are solving the main spec, not depSpec here.
 		sr = newSolveRequest(withSpec(ctx, t, spec), withBuildContext(ctx, t, workerCfg.ContextName, worker), withBuildTarget(targetCfg.Container))
-		res := solveT(ctx, t, gwc, sr)
-		ref, err := res.SingleRef()
-		if err != nil {
-			t.Fatal(err)
-		}
-
-		// Since we also added the dep as a runtime dep, the file in the base package should be installed in the output container.
-		_, err = ref.StatFile(ctx, gwclient.StatRequest{Path: "/usr/share/doc/" + depSpec.Name + "/hello.txt"})
-		if err != nil {
-			t.Fatal(err)
-		}
+		solveT(ctx, t, gwc, sr)
```

Comment:
q: why is this deleted, the `res.SingleRef()` and err handling? Shouldn't we still do that?

Reply:
I don't think that's necessary.
If there's a problem with the build `solveT` will get it.
We don't need to check the actual content of the resulting image since this test is no longer making the assumption that the worker repos will be available at install time.

Reply:
This is mentioned in the commit where the change is at, btw.

Reply:
oh ok thanks, yeah, I reviewed the whole PR